test/ft/exceptions.cpp

```diff
@@ -61,12 +63,11 @@ test exception_no_transition = [] {
   expect(sm.is(sml::X));
 };
 
-#if !defined(_MSC_VER)
 test exception_data_minimal = [] {
   struct c {
     auto operator()() const {
       using namespace sml;
-      auto guard = [](const auto& ex) { return ex.value == 42; };
+      auto guard = [](const auto &ex) { return ex.value == 42; };
```

Comment:
looks like clang-format had not been run on this line of code, do you want me to add a separate formatting commit?